internal/atls/atls.go

```diff
@@ -248,7 +250,7 @@ func verifyEmbeddedReport(validators []Validator, cert *x509.Certificate, peerPu
 				return nil
 			}
 			// Otherwise, we'll keep track of the error and continue with the next validator.
-			retErr = errors.Join(retErr, fmt.Errorf("validator %s failed: %w", validator.OID(), validationErr))
+			retErr = errors.Join(retErr, fmt.Errorf(" validator %s failed: %w ||", validator.String(), validationErr))
```

Comment:
```suggestion
			retErr = errors.Join(retErr, fmt.Errorf(" validator %s failed: %w", validator.String(), validationErr))
```

Reply:
Imo, the double dash serves better readability of the joined error, because we have no specific formatting implemented for now. But i can drop this, if it seems unnecessary.

Reply:
errors are already joined using newlines: https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/errors/join.go;l=52-54
adding custom formatting here isn't idiomatic